src/frontend/typer/behandling.ts

```diff
@@ -48,9 +48,10 @@ export enum BehandlingResultat {
 }
 
 export enum BehandlerRolle {
-    SYSTEM,
-    SAKSBEHANDLER,
-    BESLUTTER,
+    SYSTEM = 0,
```

Comment:
Hva brukes `SYSTEM` til?

Reply:
Det er egentlig bare i bruk i backend tror jeg (en SB vil aldri ha denne rollen, men enkelt ting kan være utført av systemet). Men tror ikke vi får brukt den frontend, fordi det eneste som settes backend er hvem som har gjort hva - og da settes VL for system.